### DISCUSSION TITLE
Review process

### DISCUSSION THREAD

**Discussion by sebplorenz** [International-Data-Spaces-Association/IDS-RAM_4_0#115] [Q&A] [answered]
Hi,

what is the best way to review the RAM 4 updates if I do not want to crawl through source files and pull requests? 

Regards, Sebastian

**Comment by ssteinbuss** [comment, marked as answer]
Hi @sebplorenz ,
the working phase for RAM 4.0 will end at end of February. Then we will provide a comprehensive draft. You will be asked for your review then. The final review will then be done in the upcoming meeting of the Working Group Architecture in March. 

In short: We will provide a draft version end of February. Then you can review.

**Reply by ssteinbuss** [reply]
@gaertnern from your request on the winterdays. Here your find the current state of our work https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0
